examples/semantic_segmentation.ipynb

```diff
@@ -0,0 +1,1266 @@
+{
```

Comment:
<p>Change "Here's a call for actions" to something like...</p><p> </p><p>If you want to explore further beyond this notebook, here are some things you can try next:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
Sounds way better. Thanks.